### PR TITLE
[shortcut] c+s+y open project viewer on Win OS

### PR DESCRIPTION
**Pull request by smallfont00**
this should fix issue #133

### CODE REVIEW

**Comment by dannypsnl** [comment]
Also update document and CHANGELOG

File: tool/frame.rkt [hunk 1]
```diff
@@ -61,9 +61,11 @@
                               (send current-project set path)
                               (show-real-area)
                               (send c set-label "Hide the Project Viewer"))])))]
-               ;;; c+y open project viewer
+               ;;; c+y open project viewer (c+s+y when using Windows OS)
```
Comment (dannypsnl) [inline tool/frame.rkt]: c + y
c + s + y

**Review by dannypsnl — CHANGES_REQUESTED** [review]

File: tool/frame.rkt (hunk 1 above)
Comment (smallfont00) [reply tool/frame.rkt]: are you sure? adding space look ugly to me.

Comment (dannypsnl) [reply tool/frame.rkt]: Oh, I mean change line

Comment (smallfont00) [reply tool/frame.rkt]:
```
;;; c+y on Linux, MacOS \n
;;; c+s+y on Windows \n
```
do you mean something like that?

**Review by dannypsnl — APPROVED** [review]

**Pull request merged**